docs/frequently-asked-questions.md

```diff
@@ -20,3 +20,10 @@ to keep a consistent user experience and behavior.
 
 It's worth noting that not all controllers are needed by Karmada, for the recommended controllers please
 refer to [Recommended Controllers](./userguide/configure-controllers.md#recommended-controllers).
+
+
+## Can I install Karmada in a existing kubernetes cluster as host cluster?
```

Comment:
I guess you want to explain `Can I install Karmada in an existing cluster and reuse the `kube-apiserver` as Karmada apiserver?` Am I right?

Reply:
yes